src/main/java/mate/academy/springboot/swagger/service/util/PageRequestServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.springboot.swagger.service.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PageRequestServiceImpl implements PageRequestService {
+    private static final int DIRECTION_INDEX = 1;
+    private static final int FIELD_INDEX = 0;
+
+    @Override
+    public PageRequest getPageRequest(int page, int count, String property) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (property.contains(":")) {
+            String[] sortingFields = property.split(";");

```

Comment:
Consider using a constant for the semicolon ';' character to improve readability. Example: private static final String SEMICOLON_SEPARATOR = ";"; 😊